presto-main/src/test/java/com/facebook/presto/execution/MockRemoteTaskFactory.java

```diff
@@ -141,6 +143,11 @@ public MockRemoteTask createRemoteTask(
         return new MockRemoteTask(taskId, fragment, node.getNodeIdentifier(), executor, scheduledExecutor, initialSplits, partitionedSplitCountTracker);
     }
 
+    @Override
+    public void destroyExchangeSources(Multimap<TaskId, URI> locationsToDestroy, Consumer<PrestoException> onFailure)
+    {
+    }
```

Comment:
I was thinking that your `MockRemoteTaskFactory` maintain a map from URI to `MockRemoteTask`. This way, you will be able to call `task.outputBuffer.abort(bufferId)`.

I made the argument last time that people using this test class might not realize that this isn't properly implemented. But on the other hand, I now realize that implementing this would mean code that's not meaningfully exercised, which is as bad. I think it's ok to leave this unimplemented, but maybe add a comment/TODO.